containers/Dockerfile.fedora

```diff
@@ -1,19 +1,21 @@
 FROM fedora
```

Comment:
It may be a good idea to specify `fedora:29` instead of leaving it at `fedora:latest`. This way, we know exactly which fedora version is being used. This makes more sense with Fedora's 6-month release cycles.

This then allows for time to make sure the next fedora version docker image works fine.

Reply:
Good idea. I'll pin to Fedora 29.